sdk/monitor/azure-monitor-query/tests/async/test_logs_client_async.py

```diff
@@ -47,20 +47,68 @@ async def test_logs_batch_query():
         LogsQueryRequest(
             query="AzureActivity | summarize count()",
             timespan="PT1H",
-            workspace= os.environ['LOG_WORKSPACE_ID']
+            workspace_id= os.environ['LOG_WORKSPACE_ID']
         ),
         LogsQueryRequest(
             query= """AppRequests | take 10  |
                 summarize avgRequestDuration=avg(DurationMs) by bin(TimeGenerated, 10m), _ResourceId""",
             timespan="PT1H",
-            workspace= os.environ['LOG_WORKSPACE_ID']
+            workspace_id= os.environ['LOG_WORKSPACE_ID']
         ),
         LogsQueryRequest(
             query= "AppRequests | take 2",
-            workspace= os.environ['LOG_WORKSPACE_ID']
+            workspace_id= os.environ['LOG_WORKSPACE_ID']
         ),
     ]
     response = await client.batch_query(requests)
 
     assert len(response.responses) == 3
 
+@pytest.mark.live_test_only
+@pytest.mark.asyncio
+async def test_logs_single_query_additional_workspaces_async():
+    credential = _credential()
+    client = LogsQueryClient(credential)
+    query = "union * | where TimeGenerated > ago(100d) | project TenantId | summarize count() by TenantId"
+
+    # returns LogsQueryResults 
+    response = await client.query(
+        os.environ['LOG_WORKSPACE_ID'],
+        query,
+        additional_workspaces=[os.environ["SECONDARY_WORKSPACE_ID"]],
+        )
+
+    assert response is not None
```

Comment:
You only need assert response?

Reply:
the below assert statement tests the number of rows is 2 - 1 extra row for the additional workspace

Reply:
I meant we only need "assert response" rather than "assert response is not None"? :)

Reply:
oh yes :) - i'll update it

Reply:
```suggestion
    assert response
```